pegelhub-icc-connector/pom.xml

```diff
@@ -42,5 +42,34 @@
             <version>${logback.version}</version>
         </dependency>
     </dependencies>
-
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.3.0</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addClasspath>true</addClasspath>
+                            <mainClass>com.stm.pegelhub.Main</mainClass>
+                        </manifest>
+                    </archive>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
```

Comment:
Wofür genau wird das verwendet?
Beziehungsweise brauchen wir das unbedingt?

Reply:
ohne dem kann man die jar nicht auführen weil dependencies fehlen, durch das plugin werden die dep. in die jar mit reingenommen
haben wir im FTP connector auch

Reply:
Ah ok macht natürlich Sinn.
Gut zu wissen und danke für die Erklärung :)